rootbeerlib/build.gradle.kts

```diff
@@ -77,73 +53,50 @@ dependencies {
     testImplementation(libs.mockito)
 }
 
-// helper method to ensure we have a non null string for a property
-fun getPropertyOrEmpty(propertyName: String): String = project.findProperty(propertyName)?.toString().orEmpty()
-
-project.version = getPropertyOrEmpty("VERSION_NAME")
-project.group = getPropertyOrEmpty("GROUP")
-
-fun isReleaseBuild(): Boolean = !getPropertyOrEmpty("VERSION_NAME").contains("SNAPSHOT")
-
-fun getReleaseRepositoryUrl(): String = getPropertyOrEmpty("RELEASE_REPOSITORY_URL")
-
-fun getSnapshotRepositoryUrl(): String = getPropertyOrEmpty("SNAPSHOT_REPOSITORY_URL")
-
-fun getRepositoryUsername(): String = getPropertyOrEmpty("NEXUS_USERNAME")
-
-fun getRepositoryPassword(): String = getPropertyOrEmpty("NEXUS_PASSWORD")
-
 publishing {
     publications {
         register<MavenPublication>("release") {
-            groupId = getPropertyOrEmpty("GROUP")
```

Comment:
No need to define `groupId` and `version` here - they are picked up for the definitions on the `Project` object, as set in the root `build.gradle.kts` file.